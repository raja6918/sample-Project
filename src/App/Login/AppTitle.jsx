import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const AppTitleContainerWrapper = styled.div`
  & h2 {
    font-size: 36px;
    color: #ffffff;
    margin-top: 0;
    margin-bottom: 5px;
    font-weight: 400;
  }
  & p {
    font-size: 20px;
    font-weight: 200;
    color: #ffffff;
    margin: 0;
    opacity: 0.75;
  }
`;

// eslint-disable-next-line
const shortHash = COMMITHASH.substr(0, 7);

const AppTitleContainer = ({ t }) => {
  return (
    <AppTitleContainerWrapper>
      <h2>{t('LOGIN.appTitle')}</h2>
      <p title={`Build ${shortHash}`}>{t('LOGIN.appVersion')}</p>
    </AppTitleContainerWrapper>
  );
};

AppTitleContainer.propTypes = {
  t: PropTypes.func.isRequired,
};

export default AppTitleContainer;
