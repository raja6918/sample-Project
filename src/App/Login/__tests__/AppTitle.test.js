import React from 'react';
import { mount } from 'enzyme';

import AppTitle from '../AppTitle';

const t = jest.fn();
test('AppTitle Component renders', () => {
  expect(mount(<AppTitle t={t} />)).toMatchSnapshot();
});
