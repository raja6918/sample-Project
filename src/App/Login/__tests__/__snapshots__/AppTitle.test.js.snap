// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle Component renders 1`] = `
.c0 h2 {
  font-size: 36px;
  color: #ffffff;
  margin-top: 0;
  margin-bottom: 5px;
  font-weight: 400;
}

.c0 p {
  font-size: 20px;
  font-weight: 200;
  color: #ffffff;
  margin: 0;
  opacity: 0.75;
}

<AppTitleContainer
  t={
    [MockFunction] {
      "calls": Array [
        Array [
          "LOGIN.appTitle",
        ],
        Array [
          "LOGIN.appVersion",
        ],
      ],
      "results": Array [
        Object {
          "type": "return",
          "value": undefined,
        },
        Object {
          "type": "return",
          "value": undefined,
        },
      ],
    }
  }
>
  <AppTitle__AppTitleContainerWrapper>
    <div
      className="c0"
    >
      <h2 />
      <p
        title="Build 1234567"
      />
    </div>
  </AppTitle__AppTitleContainerWrapper>
</AppTitleContainer>
`;
